website/docs/reference/dbt-jinja-functions/model.md

```diff
@@ -42,6 +42,48 @@ If you're using the CLI, use [log()](/reference/dbt-jinja-functions/log) to prin
 
 </Tabs>
 
+## Batch properties for microbatch models
+
+From dbt Core v1.9, the model object includes a `batch` property (`model.batch`), which provides details about the current batch when executing an [incremental microbatch](/docs/build/incremental-microbatch) model. This property is only populated during the batch execution of a microbatch model.
+
+The following table describes the properties of the `batch` object. Note that dbt appends the property to the `model` and `batch` objects. 
+
+| Property | Description | Example |  
+| -------- | ----------- | ------- |
+| `id` | The unique identifier for the batch within the context of the microbatch model. | `model.batch.id` |
+| `event_time_start` | The start time of the batch's [`event_time`](/reference/resource-configs/event-time) filter (inclusive). | `model.batch.event_time_start` |
+| `event_time_end` | The end time of the batch's `event_time` filter (exclusive). | `model.batch.event_time_end` |
+
+### Usage notes
+
+`model.batch` is only available during the execution of a microbatch model batch. Outside of the microbatch execution, `model.batch` is `None`, and its sub-properties aren't accessible.
+
+#### Example: Safeguard access to batch properties
```

Comment:
Will the : impact the links at all? Maybe we could use the same [style we use elsewhere](https://docs.getdbt.com/docs/collaborate/explore-projects#example-of-lenses) when `Example` is in the header 
```suggestion
#### Example of Safeguard access to batch properties
```